content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -124,7 +124,8 @@ different Kubernetes components.
 | `HPAContainerMetrics` | `false` | Alpha | 1.20 | |
 | `HPAScaleToZero` | `false` | Alpha | 1.16 | |
 | `HugePageStorageMediumSize` | `false` | Alpha | 1.18 | 1.18 |
-| `HugePageStorageMediumSize` | `true` | Beta | 1.19 | |
+| `HugePageStorageMediumSize` | `true` | Beta | 1.19 | 1.21 |
+| `HugePageStorageMediumSize` | `true` | GA | 1.22 | - |
```

Comment:
This second line should be added to the graduated/deprecated table below.

Reply:
This has been fixed in 372d34dc9467c9448a8e6a08a92e6b94260eeac5

Reply:
I've added all 3 lines as it's done for HugePages feature.

Reply:
These lines are for a GA'ed feature, so they are supposed to be MOVED to the second table, not just COPIED.

Reply:
@tengqm Thank you for pointing out to it. Done.